src/components/side-panel/layer-panel/color-selector.js

```diff
@@ -136,11 +140,18 @@ class ColorSelector extends Component {
           ))}
         </InputBoxContainer>
         {currentEditing ? (
-          <StyledPanelDropdown className="color-selector__dropdown">
+          <StyledPanelDropdown
+            className="color-selector__dropdown"
+            style={{overflow: 'auto'}}
```

Comment:
is this style necessary?

Reply:
the styling is from Prettier default. If you have a preferred custom Prettier config with specified line width and other settings, I'd like to refer to that.

Reply:
i meant
style={{overflow: 'auto'}}